host/src/prover/execution.rs

```diff
@@ -16,18 +16,22 @@ use tracing::{info, warn};
 use super::error::Result;
 use crate::{
     host::host::preflight,
+    memory,
     prover::{error::HostError, request::ProofRequest},
 };
 
 pub async fn execute<D: Prover>(config: &serde_json::Value) -> Result<Proof> {
     println!("- {:?}", config);
 
     // Generate the input
+    memory::reset_stats();
```

Comment:
I was gonna unify ur `memory` and `Measurement` together under the same struct, but realized that u put`Measurement` in `lib`.
Should have catch that but in the other PR but I think this causes lots of complication, why don't we put all stats in hosts so that no flags needed?
https://github.com/taikoxyz/raiko/blob/b549d6cb0ed6649669c91790914145c0ca486bcb/lib/src/lib.rs#L92

Reply:
The performance measurements are also done in guest code code, so not really possible to put everything in host I think. The memory stuff could be put in the lib, but because of the global allocator it seems to make things a bit more complicated because the allocator will have to be passed into the memory functions.

I guess it depends on how much and where these functions will be used, I tried to keep it very minimal and dumb for now.